index.js

```diff
@@ -79,6 +79,46 @@
 
   var _ = R.__;
 
+  /* istanbul ignore next */
+  var Any = function Any() {};
+
+  var curry = function(name, typePairs, f) {
+    return R.arity(typePairs.length, function() {
+      var args = arguments;
+      var $typePairs = [];
+      for (var idx = 0; idx < args.length; idx += 1) {
+        var typePair = typePairs[idx];
+        if (args[idx] === _) {
```

Comment:
I'd like to wait until Ramda v0.15.0 is available so that I can update this to:

```javascript
if (args[idx] != null && args[idx]['@@functional/placeholder'] === true) {
```

See ramda/ramda#1156 for details.